### ISSUE TITLE
Clustered Standard Errors in linearmodels.system.model.IVSystemGMM

### ISSUE THREAD

**Issue by lrsppp** [bashtage/linearmodels#373]
Can you estimate clustered standard errors using `linearmodels.system.model.IVSystemGMM`?

**Comment by bashtage** [comment]
Yes. See https://bashtage.github.io/linearmodels/iv/examples/advanced-examples.html#Multiple-Instruments

**Comment by lrsppp** [comment]
Thank you for your quick answer. Unfortunately, it seems not to work with `IVSystemGMM`, see:

> system_gmm = IVSystemGMM.from_formula(equations, data, weight_type="unadjusted")
> system_gmm.fit(cov_type = 'clustered')

It throws a key error: `KeyError: 'clustered'`.

**Comment by bashtage** [comment]
Oh, sorry.

**Comment by bashtage** [comment]
I just saw the IV and didn't pay attention to the System.

**Comment by bashtage** [comment]
The covariance estimators are listed here: https://bashtage.github.io/linearmodels/system/mathematical-formula.html#covariance-estimation

No clustered.

**Comment by lrsppp** [comment]
Ah, okay. Was hoping for a work around or something! Thank you!
